src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift

```diff
@@ -125,7 +125,7 @@ class ExposureSubmissionTestResultViewModel {
 	}
 	
 	func updateWarnOthers() {
-		self.warnOthersReminder.evaluateNotificationState(testResult: self.testResult)
+		warnOthersReminder.evaluateNotificationState(testResult: testResult)
 	}
```

Comment:
inside the prive section - make it private too or move?

Reply:
Moved. Thanks.